accounts/permissions.py

```diff
@@ -0,0 +1,14 @@
+from rest_framework import permissions
+
+
+class IsOwner(permissions.BasePermission):
+    """ 
+    custom permission to only allow owners of object to edit, delete or update
+    """
+
+    def has_object_permission(self, request, view, obj):
+        # allow GET methods
+        if request.method in permissions.SAFE_METHODS:
+            return True
+        # PUT, DELETE only allowed to the owner
+        return obj.owner == request.user
```

Comment:
why not bind it to the user account instead?

Reply:
> why not bind it to the user account instead?

It falls back to the first question you asked, I am relating the owner of the post and the user model

Reply:
But a post should be related to an account...?? Or what's happening.

Reply:
What is the logic to be used here? How will we identify and associate the user writing the blog content? I am still having issues with the relationship between the user and the account. 
when you log in where do you go?